flycheck.el

```diff
@@ -8852,7 +8894,7 @@ current versions of MRI and JRuby, but may break when used with
 other implementations or future versions of these
 implementations.
 
-Please consider using `ruby-rubocop' or `ruby-rubylint' instead.
+Please consider using `ruby-rubocop' or `ruby-reek' instead.
```

Comment:
I'm planing to take a look on Ruby checkers order again, when new chaining gets close to release.